substrate/frame/contracts/src/lib.rs

```diff
@@ -247,8 +247,7 @@ pub mod pallet {
 		type Randomness: Randomness<Self::Hash, BlockNumberFor<Self>>;
 
 		/// The fungible in which fees are paid and contract balances are held.
-		type Currency: Inspect<Self::AccountId>
-			+ Mutate<Self::AccountId>
+		type Currency: Mutate<Self::AccountId>
```

Comment:
Why was this change made? Seems totally independent

Reply:
I think it was a useless line as `Inspect` is a subtrait to `Mutate`